velox/common/memory/MemoryPool.cpp

```diff
@@ -770,7 +770,8 @@ bool MemoryPoolImpl::maybeIncrementReservationLocked(uint64_t size) {
       // This memory pool has been aborted by the memory arbitrator. Abort to
```

Comment:
Consider to put this into a utility
```
// comment..
void checkIfAborted() {
  if (FOLLY_UNLIKELY(aborted())) {
    VELOX_CHECK_NOT_NULL(abortError_);
    std::rethrow_exception(abortError_);
  }
}
```

Reply:
This code is only used once in code paths that do not directly have access to the error. Do we still want to put it in utility?

Reply:
It is a private method?